altiProfil/install/install.php

```diff
@@ -15,6 +15,11 @@ class altiProfilModuleInstaller extends \Jelix\Installer\Module\Installer
 {
     public function install(Jelix\Installer\Module\API\InstallHelpers $helpers)
     {
-
+        // Copy configuration file
```

Comment:
all of this are already done into the configure.php script, so no need here. You juste have to fix the path of www into configure.php